client/src/components/Token.vue

```diff
@@ -1,78 +1,62 @@
 <template>
   <div>
+    <h1>{{ getUsersName }}</h1>
     <TokensLeft/>
     <TokenChoices :cost='pickCost' choice='pick'/>
     <br/>
     <TokenChoices :cost='banCost' choice='ban'/>
 
-    <button @click='singleSubmit'>Next</button>
+    <button v-if='currPlayer' @click='previousPlayer'>Previous</button>
+    <button @click='nextPlayer'>Next</button>
   </div>
 </template>
 
 <script>
 import TokensLeft from './Token-TokensLeft.vue'
 import TokenChoices from './Token-Choices.vue'
-// import { playerChoiceStore } from '../store/player-choices.js'
+import { playerChoiceStore } from '../store/player-choices.js'
 import { EventBus } from '../main.js'
 
 export default {
-  props: ['decisionsMade'],
   data () {
     return {
       pickCost: 1,
       banCost: 1,
-      numPicks: 0,
-      numBans: 0
+      currPlayer: 0
     }
   },
   components: {
     TokensLeft,
     TokenChoices
   },
   computed: {
-    // adjust num tokens given to each player
-    totalTokenCalculation: () => {
-
-      // change 10
-      // settings.commit('setTotalTokens', 10)
-    },
-    // adjust pick/ban costs here
-    choiceCostCalculation: () => {
-
+    getUsersName () {
+      return playerChoiceStore.state[Object.keys(playerChoiceStore.state)[this.currPlayer]].name
```

Comment:
This is a long line to read. Can you break this down to individual extractions so it's easier to know what's going on?
Ex)
```js
const x = Object.keys(playerChoiceStore.state)
const y = x[this.currPlayer]
const z = playerChoiceStore.state[y]
return z.name
```